nf-test.config

```diff
@@ -0,0 +1,24 @@
+config {
+    // location for all nf-test tests
+    testsDir "."
+
+    // nf-test directory including temporary files for each test
+    workDir System.getenv("NFT_WORKDIR") ?: ".nf-test"
+
+    // location of an optional nextflow.config file specific for executing tests
+    configFile "tests/nextflow.config"
+
+    // ignore tests coming from the nf-core/modules repo
+    ignore 'modules/nf-core/**/*', 'subworkflows/nf-core/**/*'
```

Comment:
```suggestion
    ignore 'modules/nf-core/**/tests/*', 'subworkflows/nf-core/**/tests/*'
```

Reply:
I think this change deviates from the nf-core/tools template, maybe we can fix [upstream there instead](https://github.com/nf-core/tools/blob/5e54606219ad9dff4c3505900057eabfd56daee1/nf_core/pipeline-template/nf-test.config#L12)?

Reply:
It is fixed upstream already, which is why I propose to fix it there.
https://github.com/nf-core/tools/blob/dev/nf_core/pipeline-template/nf-test.config#L12

Reply:
Oh I missed this had been fixed on `dev`, I ended up pushing forward with the release in the context of your comment below plus the time constraints we're facing and knowing this fix will be applied in the next nf-core/tools template release